client/.storybook/brightmodeTheme.js

```diff
@@ -0,0 +1,11 @@
+import { create } from '@storybook/theming/create';
```

Comment:
you can create a theme in storybook but you can also use the theme you created with/for emotion with importing it. maybe look in this pull request where Susanne did it: https://github.com/Susanne1307/foos-buddy/pull/16/files
this way you don't have repetitive code... 😉

Reply:
I'*m not sure if this works for both my themes. :/